src/utils/digitalAssets/tests/mergeDigitalAssets.test.js

```diff
@@ -0,0 +1,109 @@
+// @flow strict
+
+import mergeDigitalAssets from '../mergeDigitalAssets'
+
+const JNT_SYMBOL: string = 'JNT'
+const JNT_NAME: string = 'Jibrel Network Token'
+const JNT_ADDRESS: AssetAddress = '0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7'
+
+const AZBIT_ADDRESS: AssetAddress = '0xeccab39acb2caf9adba72c1cb92fdc106b993e0b'
+const AZBIT_ADDRESS_CS: AssetAddress = '0xEccAB39acB2CAF9adBa72C1CB92FDC106B993E0b'
+
+const AZBIT_ASSET: DigitalAsset = {
+  blockchainParams: {
+    type: 'erc-20',
+    address: AZBIT_ADDRESS_CS,
+    decimals: 6,
+  },
+  symbol: 'AZ',
+  name: 'Azbit',
+  isActive: true,
+  isCustom: true,
+}
+
+describe('mergeDigitalAssets', () => {
+  test('defined', () => {
+    expect(mergeDigitalAssets).toBeDefined()
+  })
+
+  test('default items are merged correctly', async () => {
+    const result: DigitalAssets = await mergeDigitalAssets({})
+    const jnt: ?DigitalAsset = result[JNT_ADDRESS]
+
+    if (!jnt) {
+      throw new Error('JNT asset not found')
```

Comment:
No need to throw here.

First, you are testing `jnt` existence in next assertion.

Second, if test continues execution on assertion not passed, you could use safe assertions after, for example: https://jestjs.io/docs/en/expect#tohavepropertykeypath-value

Reply:
`const eth: ?DigitalAsset = result[ETH_ADDRESS]` means that `eth` can be `null` or `undefined`.
`expect(eth).toBeDefined()` will not throw if `eth` is `null`.